1.10/deploying-jobs/quickstart.md

```diff
@@ -36,18 +36,21 @@ Check the **Run on a Schedule** to reveal the following fields.
 ### **Labels**
 **Label Name** and **Label Value** - Attach metadata to your jobs so you can filter them. [Learn more about labels](/docs/1.10/tutorials/task-labels/).
 
+## Job Groups
+You can add a job to a an existing job group or create one when you create the job. Use dots in your job ID to nest the job in a group. For instance, if you enter job ID `marketing.myjob`, `myjob` will be created in the `marketing` group. In DC/OS Enterprise, you can [use job groups to implement fine-grained user access](https://docs.mesosphere.com/1.10/deploying-jobs/job-groups).
+
```

Comment:
In DC/OS Enterprise, you can use [job groups](https://docs.mesosphere.com/1.10/deploying-jobs/job-groups)  to implement fine-grained user access.